### PR TITLE
unbound PersistentVolumeClaims Postgres FAQ

### PR DESCRIPTION
**Pull request by ouyangshixiong**
FAQ tell users how to deploy activiti cloud on kubernetes( by kubeadm, on Centos7, single node cluster )

### CODE REVIEW

File: getting-started/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+spec:
+  capacity:
+    storage: 10Gi
+  accessModes:
+  - ReadWriteOnce
+  hostPath:
+    path: /data/postgres_data
+```
+
+Then open postgres yml config file(using dashboard), remove one line:   "subPath":"postgresql-db", then click update button.
```
Comment (vga16) [inline getting-started/FAQ.md]: Thanks for this solution. I cannot find "subPath":"postgresql-db". Which specific "postgres yml config file" is it? Thanks!

Comment (salaboy) [reply getting-started/FAQ.md]: @ouyangshixiong can you please help us with this question?

**Comment by salaboy** [comment]
@igdianov look at this PR is already creating a FAQ section.

File: getting-started/FAQ.md (hunk 1 above)
Comment (vga16) [reply getting-started/FAQ.md]: There is a solution..... The problem is, the AWS, GCP, etc. provide persistent storage which will be bound via the persistent storage claims. Having your non-AWS/GCP, for example using kubeadm/minikube, you have to provide those two persistent volumes 'yourself' ensuring the match the respective activity 7 PV claims originated in rabbitmq and postgres. If this is the case, activiti 7 runs smoothly! Let me know if anybody needs some more information.

Comment (ouyangshixiong) [reply getting-started/FAQ.md]: Sorry for reply so late. Some guy asked me the same question about  "subPath":"postgresql-db" , I didn't face such a problem on my local k8s cluster. After several day ,he told me that: "subPath":"postgresql-db" is not necessary, I think you can just ignore it. @salaboy @vga16

**Comment by mteodori** [comment]
obsolete, I don't think PVC are an issue on linux anymore and this is a generic unrelated issue

**Pull request closed without merging**